ui/page/overview/sync_details.go

```diff
@@ -135,7 +136,11 @@ func (pg *AppOverviewPage) progressStatusRow(gtx C, inset layout.Inset) D {
 	}
 
 	percentageLabel := pg.Theme.Body1(fmt.Sprintf("%v%%", progress))
-	timeLeftLabel := pg.Theme.Body1(fmt.Sprintf("%v left", timeLeft))
+	if progress == 0 {
+		timeLeftLabel = pg.Theme.Body1("Estimating")
```

Comment:
```suggestion
		timeLeftLabel = pg.Theme.Body1("estimating")
```

Reply:
Hello, after meeting with @raedah we've decided to replace `Estimating` with `Connecting` please.